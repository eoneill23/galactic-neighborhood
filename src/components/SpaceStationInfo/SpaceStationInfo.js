import React from 'react';
import './SpaceStationInfo.css';

const SpaceStationInfo = ({ lat, long, velo, altitude }) => {

  return (
    <section>
      <p>The ISS's current latitude is {lat}˚</p>
      <p>The ISS's current longitude is {long}˚</p>
      <p>The ISS is currently traveling at {velo} miles per hour.</p>
      <p>the ISS is currently {altitude} miles above Earth.</p>
    </section>
  )
}

export default SpaceStationInfo;