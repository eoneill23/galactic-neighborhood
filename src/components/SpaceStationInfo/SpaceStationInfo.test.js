import React from 'react';
import { shallow } from 'enzyme';
import SpaceStationInfo from './SpaceStationInfo';

describe('SpaceStationInfo', () => {

  it('should match the snapshot with all data passed in correctly', () => {

    const wrapper = shallow(<SpaceStationInfo 
      lat={-12.306527427889}
      long={-124.16292183888}
      velo={27556.87}
      altitude={423.78}
    />);

    expect(wrapper).toMatchSnapshot();
  });
});