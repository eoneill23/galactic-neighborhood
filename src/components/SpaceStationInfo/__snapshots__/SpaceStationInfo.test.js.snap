// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SpaceStationInfo should match the snapshot with all data passed in correctly 1`] = `
<section>
  <p>
    The ISS's current latitude is 
    -12.306527427889
    ˚
  </p>
  <p>
    The ISS's current longitude is 
    -124.16292183888
    ˚
  </p>
  <p>
    The ISS is currently traveling at 
    27556.87
     miles per hour.
  </p>
  <p>
    The ISS is currently 
    423.78
     miles above Earth.
  </p>
</section>
`;
